replicated-security/duality-1/join-rs-duality-cv.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -ex
+
+# Set up a Cosmovisor service to join the duality chain.
+
+# Configuration
+# You should only have to modify the values in this block
+# * Keys
+#    The private validator key and node key operations must match those used in the provider chain if you want to run a validator.
+# ***
+PRIV_VALIDATOR_KEY_FILE=~/priv_validator_key.json
+NODE_KEY_FILE=~/node_key.json
+NODE_HOME=~/.duality
+NODE_MONIKER=duality-1
+SERVICE_NAME=cv-duality
+# ***
+
+CHAIN_BINARY='dualityd'
+CHAIN_ID=duality-1
+PERSISTENT_PEERS="b7d0bd260fca7a5a19c7631b15f6068891faa60e@duality-apple.rs-testnet.polypore.xyz:26656"
```

Comment:
Can we do that before starting the network?

Reply:
Yes you should provider peers that *will* exist. One of the main lessons learned from launching other chains is that when validators are first getting set up there is a serious lack of peers available. Not only should the chain team be running seeds and peers, you should also make sure you can accept enough connections so that the p2p network can come up

Reply:
OK, this is good to note.